packages/config-driver-ssm/src/index.ts

```diff
@@ -30,66 +30,45 @@ export const parameterStoreConfigDriver = {
         kmsConfig.region = region
 
         const output: ConfigMap = {}
-        let nextToken: string | undefined
-        do {
-            logger.debug(`Fetching configuration from AWS Parameter Store: ${path}`)
-            const result = await new Promise<SSM.GetParametersByPathResult>((resolve, reject) => {
-                const client = new SSM(ssmConfig)
-                client.getParametersByPath(
-                    {
-                        Path: path,
-                        WithDecryption: true,
-                        NextToken: nextToken,
-                    },
-                    (err, data) => {
-                        if (err) {
-                            reject(new ParameterStoreError(`Cannot get parameters by path`, err))
-                        } else {
-                            resolve(data)
-                        }
-                    },
-                )
-            })
-            if (!result.Parameters) {
-                break
-            }
-            nextToken = result.NextToken
-            logger.debug(
-                `Retrieved ${result.Parameters.length} parameters from AWS Parameter Store`,
-            )
+        for (const paramName of params) {
+            const client = new SSM(ssmConfig)
+            const { Parameter } = await client
+                .getParameter({
```

Comment:
instead of looping over params, we can probably just use .getParameters &mdash; one less API call

Reply:
Fixed